src/NodeApi/Runtime/NativeLibrary.cs

```diff
@@ -58,8 +96,28 @@ public static nint Load(string libraryName)
     /// <returns>The address of the symbol.</returns>
     public static nint GetExport(nint handle, string name)
     {
+        if (handle == 0)
+            throw new ArgumentNullException(nameof(handle));
+        if (name is null)
```

Comment:
`name` cannot be `null` here. Should we rather use the `IsNullOrEmpty`?

Reply:
I really just copied this bit from Microsoft's NativeLibrary implementation upstream.

But the null check exists in upstream's NativeLibrary because nullable variables in C# are a compiler feature, and are erased at runtime. So, yes, technically, it can still be null.

Looks like you call it through NodeJSRuntime.Import, which also isn't null checked.... I can't trace this far back enough to determine if a null can get through a public contract of any kind.

So, up to you.

Reply:
Lets avoid the empty string names. I see more issues with them rather than any benefits in this code.
Also, it should avoid the formatter complains.